sift/src/main/java/siftscience/android/Queue.java

```diff
@@ -173,6 +173,13 @@ void append(@NonNull MobileEventJson event) {
         }
     }
 
+    void forceUpload(){
```

Comment:
nit - there should be a space after the `()` like `void forceUpload() {`

Reply:
Updated